pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerGauge.java

```diff
@@ -41,6 +41,7 @@
   REALTIME_OFFHEAP_MEMORY_USED("bytes", false),
   REALTIME_SEGMENT_NUM_PARTITIONS("realtimeSegmentNumPartitions", false),
   LLC_SIMULTANEOUS_SEGMENT_BUILDS("llcSimultaneousSegmentBuilds", true),
+  RESIZE_TIME_MS("resizeTimeMs", false),
```

Comment:
This should be defined as RESIZE_TIME_MS("milliSeconds", false). See the other duration related gauge metrics.

Reply:
It actually may not matter but if you look at the constructor of ServerGauge, the unit is the first argument